projects/exui-common-lib/src/lib/exui-common-lib.module.ts

```diff
@@ -131,14 +131,15 @@ export const GOV_UI_COMPONENTS = [
     ...COMMON_COMPONENTS,
     ...GOV_UI_COMPONENTS
   ],
-  imports: [
-    CommonModule,
-    FormsModule,
-    ReactiveFormsModule,
-    RouterModule.forChild([]),
-    MatAutocompleteModule,
-    MatTabsModule
-  ],
+    imports: [
```

Comment:
Minor query - Why is there an indent here?

Reply:
IDE just correcting things.